src/3D-Slicer-1.0.xml

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+<license isOsiApproved="false" licenseId="3D-Slicer-1.0"
+         name="3D Slicer License v1.0" listVersionAdded="3.24">
```

Comment:
In the issue the approved name is "3D Slicer License". Which one we want to use?